### ISSUE TITLE
screen.width vs. max(width, height)

### ISSUE THREAD

**Issue by tomdav999** [dimsemenov/PhotoSwipe#1205]
I've noticed several instances in the code where different behavior results if screen.width exceeds some threshold.  This results in inconsistent behavior when the screen is rotated.  I would think it better to use Math.max(screen.width, screen.height) to ensure consistent behavior.

**Comment by thomasfrobieter** [comment]
Yes, this is still buggy. We set Initial Zoom Level to 0.75, what works perfectly fine on desktop, but has kind of a 1.5x initial zoom on mobile devices.
